docs/user/integration/integrate-jaeger.md

```diff
@@ -0,0 +1,181 @@
+# Install custom Jaeger in Kyma
```

Comment:
Could you please add a small table to every guide indicating the discussed signal types (logs, metrics,traces) and if the backend is in-cluster or external

Reply:
Good idea!

Reply:
Wait, I'm not completely sure what you envision @a-thaler. Pls advise.